diff -Nur old/src/frox.conf new/src/frox.conf
--- old/src/frox.conf	2004-08-13 06:32:18.000000000 +0000
+++ new/src/frox.conf	2004-09-20 20:08:59.000000000 +0000
@@ -73,8 +73,8 @@
 # you really want to run as root (not a good idea) you must say so
 # specifically, and have compiled with --enable-run-as-root.
 #
-# User nobody
-# Group nogroup
+User frox
+Group frox
 
 # This is frox's working directory - it must be specified. Temporary
 # files and sockets will be created here. If you are using local
@@ -83,7 +83,7 @@
 # also chroot to this dir on startup. To avoid this you must specifically 
 # set DontChroot to Yes.
 #
-# WorkingDir /usr/local/lib/frox
+WorkingDir /var/lib/frox
 # DontChroot Yes
 
 # Logging level. 0=No logging. 5=Critical errors only. 10= All errors.
@@ -97,7 +97,7 @@
 # below.
 #
 # LogLevel 15
-# LogFile /usr/local/lib/frox/frox-log
+# LogFile /var/log/frox/frox-log
 # XferLogging no
 
 # File to store PID in. Default is not to. If this file is not within
